### PR TITLE
Added comments for create_laplacian_2d

### PR DESCRIPTION
**Pull request by Mnlohani**
Please review the comments added for create_laplacian_2d Function. Thank you

### CODE REVIEW

File: urbanbroccoli/laplacian.py [hunk 1]
```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
     return laplacian
 
 def create_laplacian_2d(nx, ny, hx, hy):
-    laplacian = zeros(shape=(nx, ny, nx, ny), dtype=float64)
+    '''Creates Laplacian matrix for 2-D Domain by discretizing the domain by grids of required sizes in X and Y direction.
```
Comment (cwehmeyer) [inline urbanbroccoli/laplacian.py]: This line is way too long. Please shorten to just a few words which describe as basic as possible what is going on.

File: urbanbroccoli/laplacian.py [hunk 2]
```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
     return laplacian
 
 def create_laplacian_2d(nx, ny, hx, hy):
-    laplacian = zeros(shape=(nx, ny, nx, ny), dtype=float64)
+    '''Creates Laplacian matrix for 2-D Domain by discretizing the domain by grids of required sizes in X and Y direction.  
+       
+       It first creates a 4-dimensional array to show all components of laplace operator at each grid point.
```
Comment (cwehmeyer) [inline urbanbroccoli/laplacian.py]: The explanation is a bit verbose. And, please, do not use such long lines.

File: urbanbroccoli/laplacian.py [hunk 3]
```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
...
 def create_laplacian_2d(nx, ny, hx, hy):
-    laplacian = zeros(shape=(nx, ny, nx, ny), dtype=float64)
+    '''Creates Laplacian matrix for 2-D Domain by discretizing the domain by grids of required sizes in X and Y direction.  
+       
+       It first creates a 4-dimensional array to show all components of laplace operator at each grid point.  
+       In 4-D array,each inner 2-D array[nx,ny] represents array of all componants of laplacian Operator of a perticular grid point.  
+       Filling it according to Discetized laplace operator taking care of periodic boundary conditions.  
+       And then reshaping it into laplaction matrix .  
+       
+       Input Arguments:
```
Comment (cwehmeyer) [inline urbanbroccoli/laplacian.py]: Remove four spaces indention just call it ``Parameters:``

File: urbanbroccoli/laplacian.py [hunk 4]
```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
...
+       nx(integer): number of grid points in X-direction  
+       ny(integer): number of grid points in Y-direction 
+       hx(float): grid-size in X-direction
```
Comment (cwehmeyer) [inline urbanbroccoli/laplacian.py]: Grid spacing might be a better word than size

File: urbanbroccoli/laplacian.py [hunk 5]
```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
...
+       hy(float): grid-size in Y-direction
+       
+       Output:
```
Comment (cwehmeyer) [inline urbanbroccoli/laplacian.py]: Pleas remove four spaces indention and call it ``Returns:``

**Review by cwehmeyer — CHANGES_REQUESTED** [review]

File: urbanbroccoli/laplacian.py (hunk 1 above)
Comment (Mnlohani) [reply urbanbroccoli/laplacian.py]: Changes done.

File: urbanbroccoli/laplacian.py (hunk 2 above)
Comment (Mnlohani) [reply urbanbroccoli/laplacian.py]: Changes done.

File: urbanbroccoli/laplacian.py (hunk 3 above)
Comment (Mnlohani) [reply urbanbroccoli/laplacian.py]: Changes done

File: urbanbroccoli/laplacian.py (hunk 4 above)
Comment (Mnlohani) [reply urbanbroccoli/laplacian.py]: Changes done

File: urbanbroccoli/laplacian.py (hunk 5 above)
Comment (Mnlohani) [reply urbanbroccoli/laplacian.py]: Changes done

**Review by Mnlohani — COMMENTED** [review]
All changes done .Please review.

**Review by cwehmeyer — APPROVED** [review]

**Pull request merged**